nano/test_common/system.cpp

```diff
@@ -630,3 +630,33 @@ void nano::test::cleanup_dev_directories_on_exit ()
 		nano::remove_temporary_directories ();
 	}
 }
+
+std::shared_ptr<nano::election> nano::test::system::start_election (nano::node & node_a, const std::shared_ptr<nano::block> & block_a)
+{
+	deadline_set (5s);
+
+	// wait until and ensure that the block is in the ledger
+	while (!node_a.block (block_a->hash ()))
+	{
+		if (poll ())
+		{
+			return {};
```

Comment:
I think this should return nullptr to be more specific.

Reply:
OK